test/e2e/common/pods.go

```diff
@@ -235,16 +235,20 @@ var _ = framework.KubeDescribe("Pods", func() {
 		deleted := false
 		timeout := false
 		var lastPod *v1.Pod
-		timer := time.After(30 * time.Second)
+		timer := time.After(2 * time.Minute)
 		for !deleted && !timeout {
```

Comment:
Get rid of timeout variable.
This change can be made later, since this is urgent